### ISSUE TITLE
Project Settings / Font Style does not store settings

### ISSUE THREAD

**Issue by Etsija** [xulihang/ImageTrans-docs#300]
![Capture](https://user-images.githubusercontent.com/599904/193505028-7cef93f1-2090-432c-a09a-8d59133cdcdf.JPG)

I'm trying to set up a global font setting for my project, by defining a new style and naming it in Project Settings. But when I define my new style and click Add, the style is not added to the left window in Font style tab. Also the global style is NOT applied to my project, which means I need to change all translated text boxes to the new font manually, as the program wants to use System font all the time.

So in a nutshell: the left-side style list remains completely empty whatever I do. None of my defined styles are populated there, so I guess no styles are saved for my project, either.

**Comment by xulihang** [comment]
Some items are compulsory like the style'ss name, the PS font name, otherwise the style won't be added.

**Comment by Etsija** [comment]
And how do I get the PS font name, when I don't own Photoshop? Does it need to be the exact PS font name? This is just a free font downloaded from the internet.

**Comment by Etsija** [comment]
Nevermind, I got it to work, as it seems to accept any font name. This can be closed, thank you.

**Comment by xulihang** [comment]
If you do not need to use PS. you can just use any name. If you do, you can find the name according to this: https://imagetrans.readthedocs.io/en/latest/gettingstarted.html#photoshop
